python/ray/tune/tuner.py

```diff
@@ -1,5 +1,23 @@
+import traceback
+from typing import Any, Callable, Dict, Optional, Type, Union
+
+import ray
+
+from ray.ml.config import RunConfig
+from ray.ml.trainer import Trainer
+from ray.tune import TuneError
 from ray.tune.result_grid import ResultGrid
+from ray.tune.trainable import Trainable
+from ray.tune.impl.tuner_internal import TunerInternal
+from ray.tune.tune_config import TuneConfig
 from ray.util import PublicAPI
+from ray.util.client.common import ClientActorHandle
+from ray.util.ml_utils.node import force_on_current_node
+
+
+# The magic key that is used when instantiating Tuner during resume.
+_TUNER_INTERNAL = "_tuner_internal"
+_SELF = "self"
 
 
 @PublicAPI(stability="alpha")
```

Comment:
Args, not attributes (below)

Reply:
See https://github.com/ray-project/ray/pull/22975#discussion_r824311556